__tests__/app.test.js

```diff
@@ -158,6 +158,7 @@ describe("GET /api/articles/:article_id", () => {
         expect(article).toHaveProperty('created_at');
         expect(article).toHaveProperty('votes');
         expect(article).toHaveProperty('article_img_url');
+        expect(article).toHaveProperty('comment_count');
```

Comment:
It's bad practise to alter previous tests, please add a new test to check for the new property.